### PR TITLE
Snap fb text

### PR DESCRIPTION
**Pull request by sarath-yerra**

### CODE REVIEW

**Comment by davidpickering** [comment]
@sarath-yerra can you throw another commit on this PR to take this bit of code out of scope:

"View All of Our Upcoming Distributions" - 

image

Also the API response looks great

image

File: src/Modules/General/FoodbankTextComponent.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    return (       
+            <div className="row">
+                <div className="col-1 item-icon d-flex align-items-center">
+                    <img alt={text} src={imageUrl} />
+                </div>
+                <div className="col-8 item-details">
+                    <p>{text}</p>
+                </div>
+                <div style={{"word-wrap": "break-word"}} className="col-3">
+                    <a href={LinkUrl} target="_blank"> {linkText}</a>
```
Comment (maheshkurma) [inline src/Modules/General/FoodbankTextComponent.js:13]: Using target="_blank" without rel="noopener noreferrer" is a security risk
see- https://mathiasbynens.github.io/rel-noopener

File: src/Modules/General/__test__/foodbanktextComponent.test.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import FoodbankTextComponent from '../FoodbankTextComponent';
+import { mockFoodbankText } from '../../../Testing';
+
+
+test(`should display the fb_texts data`, () => {
+  const { getByText } = render(<FoodbankTextComponent text={mockFoodbankText.text} imageUrl={mockFoodbankText.image_resource} imageUrl={mockFoodbankText.link_href}/>)
+  getByText(mockFoodbankText.text);
+});
```
Comment (maheshkurma) [inline src/Modules/General/__test__/foodbanktextComponent.test.js:10]: end of line missing

File: src/Testing/mock-foodbanks/index.js [hunk 3]
```diff
@@ -11,6 +17,8 @@ const mockFoodBankBuilder = build('Food Bank').fields({
   fb_agency_locator_url: fake(f => f.internet.url()),
   fb_url: fake(f => f.internet.url()),
   fb_fano_url: fake(f => f.internet.url()),
+  foodbank_texts: [mockFoodbankTextBuilder()],
 });
 
 export const mockFoodBank = mockFoodBankBuilder();
+export const mockFoodbankText = mockFoodbankTextBuilder();
```
Comment (maheshkurma) [inline src/Testing/mock-foodbanks/index.js:24]: end of line missing

File: src/Modules/General/FoodbankTextComponent.js (hunk 1 above)
Comment (maheshkurma) [reply src/Modules/General/FoodbankTextComponent.js:13]: u can update 
from `<a href={LinkUrl} target="_blank"> {linkText}</a>`
 to `<a href={LinkUrl} target="_blank" rel="noopener noreferrer"> {linkText}</a> `

Comment (sarath-yerra) [reply src/Modules/General/FoodbankTextComponent.js]: Done added

**Pull request merged**